File: kitchen-service/src/database/connection.js
const mysql = require('mysql2/promise');
require('dotenv').config();

const pool = mysql.createPool({
  host: process.env.DB_HOST || 'localhost',
  port: process.env.DB_PORT || 3306,
  user: process.env.DB_USER || 'kitchen_user',
  password: process.env.DB_PASSWORD || 'kitchen_pass',
  database: process.env.DB_NAME || 'kitchen_db',
  waitForConnections: true,
  connectionLimit: 10,
  queueLimit: 0
});

module.exports = pool;

File: kitchen-service-python/src/database/connection.py
import mysql.connector
import os
from dotenv import load_dotenv

load_dotenv()

def get_db_connection():
    """Get MySQL database connection"""
    return mysql.connector.connect(
        host=os.getenv("DB_HOST", "localhost"),
        port=int(os.getenv("DB_PORT", 3306)),
        user=os.getenv("DB_USER", "kitchen_user"),
        password=os.getenv("DB_PASSWORD", "kitchen_pass"),
        database=os.getenv("DB_NAME", "kitchen_db"),
        autocommit=False
    )








